roles/edpm_neutron_sriov/tasks/install.yml

```diff
@@ -15,13 +15,16 @@
 # under the License.
 
 - name: Create neutron-sriov-agent directories
+  become: true
```

Comment:
+1
ideally you would also remove the become true here

https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/playbooks/neutron_sriov.yml#L6

the neutron role should elevate in the tasks that need it now at the playbook level

the same issue is present in all 4 neutron playbooks
this seams to have been cargo culted to all the neutron playbooks from the ovn playbook but since its a preexsitign bug it could be fixed in  different pr

Reply:
> +1 ideally you would also remove the become true here
> 
> https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/playbooks/neutron_sriov.yml#L6
> 
> the neutron role should elevate in the tasks that need it now at the playbook level
> 
> the same issue is present in all 4 neutron playbooks this seams to have been cargo culted to all the neutron playbooks from the ovn playbook but since its a preexsitign bug it could be fixed in different pr

Ack